packages/chat-component-bindings/src/utils/getParticipantsWhoHaveReadMessage.test.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { ChatMessageWithStatus } from '@internal/chat-stateful-client';
+import { getParticipantsWhoHaveReadMessage } from './getParticipantsWhoHaveReadMessage';
+
+const chatMessage = (messageID: string): ChatMessageWithStatus => ({
+  id: messageID,
+  type: 'text',
+  sequenceId: '89542038450',
+  version: 'jehkfhsdf',
+  createdOn: new Date(),
+  status: 'delivered'
+});
+
+const readReceiptForEachSender = {
+  '12345': { lastReadMessage: '3', name: 'c' },
```

Comment:
It would be nice to call this one lastReadMessageId in production code (let's do it when next time you change production code)

Reply:
yeah np!